packages/workers-shared/utils/types.ts

```diff
@@ -6,8 +6,14 @@ const InternalConfigSchema = z.object({
 	debug: z.boolean().optional(),
 });
 
+const StaticRoutingSchema = z.object({
+	user_worker: z.array(z.string()),
+	asset: z.array(z.string()).optional(),
```

Comment:
Where will the logic live for splitting the rules into two separate arrays?

Reply:
Isn't that where it was already handled? We need to use this splitting logic in Miniflare and the Vite plugin for dev so ideally there would be a single implementation that can be shared.

Reply:
whoops, that was not what i meant. updated to _accept_ already-split routing in EWC

Reply:
I see you've switched to keeping the splitting in EWC to make things consistent with the API. That makes sense. Out of curiosity, what's the reasoning for not having the splitting executed in the Worker e.g. a `generateStaticRoutingRuleMatchers` function that returns `{ excludeRulesMatcher, includeRulesMatcher }`?

Reply:
as mentioned async; the split is just to save a few cycles in the router worker. having the lists pre-split means the router worker has less cpu work required to split and/or traverse the entire set of paths just to find the exclude paths

hard to imagine that work mattering, but that was the intent